CONTRIBUTING.md

```diff
@@ -61,6 +61,14 @@ yarn build-editor
 yarn preview
 ```
 
+## Visual Regression
+
+To test if your changes make a difference visually across a large number of specifications, you can use can use the `yarn screenshot` command. 
+This command will use `dist/gosling.js` in a headless browser (using puppeteer) and take a screenshot of the visualization that is included in the editor. 
+It will save these screenshots in `img/visual-regression/new-screenshots`. If there is a difference compared to the screenshots in `img/visual-regression/screenshots`
```

Comment:
So, for now, would the usage pattern be like (occasionally) running the `yarn build` and `yarn screenshot` and then moving all files under `img/visual-regression/new-screenshots` to `img/visual-regression/screenshots`?